app/controllers/api/v1/road_trip_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::RoadTripController < ApplicationController
+  def create
+    user = User.find_by(api_key: road_trip_params[:api_key])
+    if user.present?
+      road_trip = RoadTrip.plan_trip(road_trip_params)
+      render json: RoadTripSerializer.new(road_trip)
+    else
+      error = 'Cannot authenticate API key'
+      render json: { error: error }, status: :unauthorized
```

Comment:
learned about rendering errors with [this article](https://www.thegreatcodeadventure.com/rails-api-painless-error-handling-and-rendering-2/)